modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/value/ValueProviderFactory.java

```diff
@@ -80,25 +77,28 @@ ValueProvider createValueProvider() throws ValueResolvingException {
     }
   }
 
-  private Optional<ValueResolvingException> injectValueProviderFields(ValueProvider resolver)
-      throws org.mule.runtime.module.extension.internal.runtime.ValueResolvingException {
+  private void injectValueProviderFields(ValueProvider resolver) throws ValueResolvingException {
     List<String> missingParameters = new ArrayList<>();
     for (InjectableParameterInfo injectableParam : factoryModelProperty.getInjectableParameters()) {
+      Object parameterValue = null;
       String parameterName = injectableParam.getParameterName();
       try {
-        Object parameterValue = parameterValueResolver.getParameterValue(parameterName);
+        parameterValue = parameterValueResolver.getParameterValue(parameterName);
+      } catch (org.mule.runtime.module.extension.internal.runtime.ValueResolvingException ignored) {
+        // no op
```

Comment:
why? very weird

Reply:
because `getParameterValue` can throw that `ValueResolvingException` if the resolution fails, but we don't want neither to fail there nor throw that exception, just to mark the parameter as missing and then throw the general exception bellow with all the info.

Reply:
that said, I'll add a debug log